locking/admin.py

```diff
@@ -116,22 +117,46 @@ def get_urls(self):
         ]
         return locking_urls + urls
 
+    def get_api_url(self, object_id):
+        app_label, model_name = self._model_info
+
+        reverse_kwargs = {
+            'app': app_label,
+            'model': model_name,
+        }
+        if object_id is not None:
+            reverse_kwargs['object_id'] = object_id
+
+        return reverse('locking:locking-api', kwargs=reverse_kwargs)
```

Comment:
This commit introduces a new required namespace for the locking API URLs. If users follow the instructions in the readme and add:

```
url(r'^locking/', include('locking.urls')),
```

To their `urls.py` file, then they will get the error `NoReverseMatch: u'locking' is not a registered namespace`. Is there a way to update `locking/urls.py` to add the namespace automatically?

If not, you'll need to:
 * [ ] update the readme
 * [ ] document this breaking change in the changelog
 * [ ] change the version number to 2.0.0 to reflect the fact that this has backwards incompatible changes